src/dials/command_line/generate_mask.py

```diff
@@ -64,6 +69,7 @@
 def generate_mask(
     experiments: ExperimentList,
     params: phil.scope_extract,
+    starting_masks: list = None,
```

Comment:
can you constrain `list = None` to be `None | list(Mask)`?

Reply:
The `|` is only python 3.10, and I assume we are still supporting 3.9? At least the end of life for 3.9 is next year.  But Union works:

```
    starting_masks: list = Union[None, Masks],
```